package.json

```diff
@@ -75,6 +75,7 @@
         "fanfou-sdk": "5.0.0",
         "form-data": "4.0.0",
         "googleapis": "134.0.0",
+        "got": "14.2.1",
```

Comment:
`got` is not a production dependency since https://github.com/DIYgod/RSSHub/pull/14922

Reply:
`pnpm build` would fail after running 'pnpm install --production` if I don't add `got` to production dependencies

Reply:
You can use test that behavior using https://github.com/wrvsrx/RSSHub/tree/only-add-tosource, this branch only add `tosource` as production dependencies.

`pnpm install --production` then `pnpm build` would fail on that branch.

To have a clean `node_modules`, you can use `pnpm prune --production`.

Reply:
I'll commit the patch to PR later.